README.md

```diff
@@ -22,6 +22,8 @@ linkerd is built on top of [Netty][netty] and [Finagle][finagle], a
 production-tested RPC framework used by high-traffic companies like Twitter,
 Pinterest, Tumblr, PagerDuty, and others.
 
+linkerd is hosted by the Cloud Native Computing Foundation ([CNCF][cncf]).
+
 ## Want to try it? ##
```

Comment:
Not part of your change, but I'd recommend also linking to the official linkerd image on dockerhub in this section. Something like:

> We also publish docker images for each release, which you can find on [docker hub](https://hub.docker.com/r/buoyantio/linkerd/).